src/i18n/it.ts

```diff
@@ -0,0 +1,33 @@
+
+import { createLocale } from './utils'
+
+export default createLocale({
+  'PROCESSING' () {
+    return 'Caricamento…' as const
+  },
+  'BTN_ERROR' () {
+    return '❌Download Fallito!' as const
+  },
+
+  'DEPRECATION_NOTICE' (btnName: string) {
+    return `Attenzione!\nSembra che \`${btnName}\` non funzioni correttamente. Usa \`Parti Singole\`.\n(Potrebbe funzionare. Clicca \`OK\` per continuare.)` as const
```

Comment:
Doesn't the translation change the original meaning?

Google Translate tells me `Warning! It seems that btnName is not working correctly. Use Single Parts (This may work. Click OK to continue.)`